SECURITY.md

```diff
@@ -0,0 +1,5 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+To report a vulnerability, create an issue with details on the vulnerability and how to exploit it, or create a pull request with a fix for the vulnerability.
```

Comment:
Opening a public issue for a security vulnerability is generally a very bad idea (https://en.wikipedia.org/wiki/Responsible_disclosure).

Disclosures like this need to be private at first. Best way to do that would probably be via https://selfhosted.show/contact.